src/world/common/DeadFoliage.inc.c

```diff
@@ -0,0 +1,38 @@
+#include "common.h"
+#include "map.h"
+
+void N(dead_foliage_setup_shear_mtx)(Matrix4f mtx, f32 scale, f32 xAmount, f32 zAmount) {
```

Comment:
Is this function being snakecase proper? I thought these scripts were UpperCamelCase.

Reply:
This isn't a script_api function

Reply:
This isn't a script API function, only those are ThisCase; not all N-funcs are for scripts!